engine-tests/src/test/java/org/terasology/documentation/ApiScraper.java

```diff
@@ -84,6 +84,7 @@ public static void main(String[] args) throws Exception {
                     } else {
                         //System.out.println("Jar-based Class: " + apiClass + ", came from " + location);
                         sortedApi.put(category, apiClass.getName() + (apiClass.isInterface() ? " (INTERFACE)" : " (CLASS)"));
+
```

Comment:
Stray newline? If only change in file reverting would make for a cleaner diff/history :-)